### ISSUE TITLE
Linked spreadsheet: Inconsistent fonts

### ISSUE THREAD

**Issue by matthijskooijman** [nextcloud/forms#1972]
I previously posted this as feedback on the PR before it was applied (https://github.com/nextcloud/forms/pull/1758#issuecomment-1896763727), but since it was out of scope for the initial PR, I'm reposting this here.

**Describe the bug**
When letting the form plugin write its results to an external spreadsheet automatically (as recently supported with merging #1758), I've noticed that the font used for form data (Calibri) is different from the default font my libreoffice uses when adding data to empty cells (Liberation Sans). I'm not at all sure how this works and if this is a matter of system-dependent defaults or not (I have not tested with collabora, since I do not have it on my test Nextcloud instance), but it is a bit unexpected.

I suspect that the forms plugin might hardcoded the font to use, while there probably is a file-wide default font that it could be using? If so, this would be a bug in the forms plugin (but maybe it's a bug in my expectations or not really a bug - I'm not entirely sure).

**To Reproduce**
1. Set up writing results to an external spreadsheet
2. Open the spreadsheet in e.g. libreoffice
3. Add some data to an empty cell
4. See the font is different from the existing data

**Expected behavior**
The data added by the forms plugin should use the same font as other data manually added.

**Screenshots**
![image](https://github.com/nextcloud/forms/assets/194491/232027b0-98cc-4e50-a054-8cdda801dc95)

**Nextcloud (please complete the following information):**
- Nextcloud-Version: 28.0.1
- Forms-Version: 4.0.0 with PR #1758 applied

**Comment by susnux** [comment]
Could you retry with forms 4.1?
We do not set any font, meaning the entries `forms` write are all in the default font (often shown as "Calibri").
If you now insert text with libreoffice it will use the font you set up as default in libre office, but keep the ODS default for the already filled cells.

**Comment by Chartman123** [comment]
@matthijskooijman did you already find some time to check susnux's comment?

**Comment by Chartman123** [comment]
No answer for quite some time, closing for now. Please feel free to re-open, after providing us with the needed info.

**Comment by matthijskooijman** [comment]
I (finally) got around to looking at this again, and it still seems something weird is going on with the spreadsheet created by Forms. I used Forms app 5.1.0.

I created a form, let that create an ODS for the results, submitted the form and opened the ODS in Collabora.
 - The rows that were created by Forms, used Calibri 11
 - A new row I add in Collabora, uses Carlito 10
 - When I download the ODS, open it with libreoffice, the row added with Collabora became Calibri 11. Adding a new row, I get Liberation Sans 10.
 - Opening up the ODS again in Collabora (without uploading), the line I added is now also Calibri (instead of Carlito as before). Maybe something weird happened in between?
 - Adding one more row with Collabora - again Carlito.
 - Downloading and opening again in libreoffice, the last added row is still Carlito. Adding a new row in Libreoffice also becomes Carlito.

For comparison, if I create a new ODS spreadsheet inside Nextcloud Web UI and open it in Collabora, add a row, I get DejaVu Sans. Downloading it to LibreOffice, I still see DejaVu Sans and new rows also use DejaVu Sans.

The latter shows that it should be possible to have an ODS file that behaves consistently.

It looks a bit like like there might be a "default font" configuration directive inside an ODS file, but the ODS generated by Forms does not set this, meaning each editor applies its own default somehow?
